_data/sidebars/user_docs.yml

```diff
@@ -24,7 +24,7 @@ entries:
       output: web, pdf
 
     - title: Bootstrap an Image
-      url: /bootstrap-image
+      url: /image-content
```

Comment:
hey @ebetica ! Why is the link above broken? The page being referenced should have permalink "bootstrap-image" - to get to thispage: https://github.com/singularityware/singularityware.github.io/blob/master/pages/docs/user-docs/docs-bootstrap-image.md. which renders to http://singularity.lbl.gov/bootstrap-image. Is the error somewhere else and we need to change a different file?